RequestKit.xcodeproj/project.pbxproj

```diff
@@ -867,6 +868,7 @@
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";
 				SKIP_INSTALL = YES;
 				SWIFT_OPTIMIZATION_LEVEL = "-Owholemodule";
+				SWIFT_VERSION = 3.0;
```

Comment:
No, you have to remove it from the target build settings instead of the project build settings.

If nothing is specified in the targets they inherit it from the project. To sync all targets (iOS, watchOS, tvOS and macOS) we need to have it in the project. The migrator only sets it in the first target it finds (iOS) so you have to manually remove it after the migration is finished.